next.config.js

```diff
@@ -1,9 +1,30 @@
 const path = require("path");
 
-module.exports = {
-  webpack: (config) => {
+/**
+ * @type {import('next').NextConfig}
+ */
+const nextConfig = {
+  webpack: (config, { isServer }) => {
     config.resolve.alias["~"] = path.resolve(__dirname);
 
+    // should only include these for testing env
```

Comment:
I think the if condition is clear that these config values are only being included for testing. Feel free to remove the comment.